### PR TITLE
Replace Lorem text in main activity with something meaningful

### PR DESCRIPTION
**Pull request by waqarsoomro9714**
completed the task "Replace Lorem text in main activity with something meaningful" changed it to take care of your loved onces.

### CODE REVIEW

**Comment by waqarsoomro9714** [comment]
please review my code thanks.

**Comment by el-veee** [comment]
Thanks for using a string resource. 

Couple points of feedback. 
- Can you rename your PR to something more descriptive of the change you've made. You could use the title of the Trello card if you like
- You have a merge conflict to resolve

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <resources>
     <string name="app_name">TG_Patient_Profile</string>
 
+    <string name="slogan_on_main_activity">Take care of your loved onces</string>
```
Comment (el-veee) [inline app/src/main/res/values/strings.xml]: I like most of this but I don't think "loved ones" makes sense for a patient management app to be used in a professional setting. 

Something like "Take Care of your Patients" is simple and to the point

**Comment by waqarsoomro9714** [comment]
please check the code thank you.

**Comment by el-veee** [comment]
You're starting to tackle a separate task in this PR. They should be separate PRs from separate branches.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <resources>
     <string name="app_name">TG_Patient_Profile</string>
 
+    <string name="slogan_on_main_activity">We take care of your loved onces</string>
```
Comment (el-veee) [inline app/src/main/res/values/strings.xml]: Again, this app is meant to be used by professionals. The "loved ones" here would be patients. This app isn't a bridge between caregivers and patient's families so I don't think this text suits Guardian

**Comment by el-veee** [comment]
You should also only tackle 1 model migration per PR. There's separate Trello cards for each of these. This is to reduce the risk of regressions

File: app/src/main/java/com/example/tg_patient_profile/model/MedicalDiagnostic.kt
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/model/MedicalDiagnostic.kt:1]: Should not be in this PR

File: app/src/main/java/com/example/tg_patient_profile/model/Medical_diagnostic.java
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/model/Medical_diagnostic.java:1]: Should not be in this PR

File: app/src/main/java/com/example/tg_patient_profile/model/NextOfKin.java
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/model/NextOfKin.java:1]: Should not be in this PR

File: app/src/main/java/com/example/tg_patient_profile/model/NextOfKin.kt
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/model/NextOfKin.kt:1]: Should not be in this PR

File: app/src/main/java/com/example/tg_patient_profile/view/general/PatientProfileAddActivity.java
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/view/general/PatientProfileAddActivity.java:1]: Should not be in this PR

File: app/src/main/java/com/example/tg_patient_profile/view/patient/patientdata/medicaldiagnostics/CurrentMedicalDiagnosticsFragment.java
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/view/patient/patientdata/medicaldiagnostics/CurrentMedicalDiagnosticsFragment.java:1]: Should not be in this PR

File: app/src/main/java/com/example/tg_patient_profile/view/patient/patientdata/medicaldiagnostics/PastMedicalDiagnosticsFragment.java
Comment (el-veee) [inline app/src/main/java/com/example/tg_patient_profile/view/patient/patientdata/medicaldiagnostics/PastMedicalDiagnosticsFragment.java:1]: Should not be in this PR

**Review by el-veee — CHANGES_REQUESTED** [review]
This PR should only contain changes related to updating the Main Activity text. You should remove the model changes, and create separate PRs for them. You can uses branching for this.

**Pull request closed without merging**